src/test/java/PrivacyPolicyTest.java

```diff
@@ -0,0 +1,49 @@
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.PrivacyPolicyPage;
+
+import java.time.Duration;
+
+public class PrivacyPolicyTest {
+
+    @Test
+    public void PrivacyPolicyTest() {
+
+        // initialisation in Icognito Mode
+        DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+        desiredCapabilities.setBrowserName("Chrome");
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--incognito");
+        desiredCapabilities.setCapability(ChromeOptions.CAPABILITY, options);
+
+        WebDriver browser = new ChromeDriver(options);
+
+        WebDriverWait wait = new WebDriverWait(browser, Duration.ofMillis(1000));
+        browser.manage().window().maximize();
```

Comment:
All settings that are needed to run the test must be done in the blocks before and after